charts/node-feature-discovery/README.md

```diff
@@ -0,0 +1,71 @@
+# deCONZ helm chart
```

Comment:
This should probably be `node-feature-discovery`

Reply:
You're probably right, fixed in latest commit 😉